lld/ELF/InputFiles.cpp

```diff
@@ -554,6 +593,7 @@ template <class ELFT> void ObjFile<ELFT>::parse(bool ignoreComdats) {
   sections.resize(size);
   for (size_t i = 0; i != size; ++i) {
     const Elf_Shdr &sec = objSections[i];
+
```

Comment:
I think this whitespace diff can be remove too.

Reply:
The blank line is still there?